assets/sass/patterns/molecules/personalised-cover-download.scss

```diff
@@ -54,27 +66,26 @@
 
 .button-collection {
   display: grid;
-  grid-row: 2;
   justify-items: center;
 }
 
-.button-collection__item {
-  @include margin(12, "left");
-  @include margin(12, "right");
-}
+ol.personalised-cover-buttons[data-behaviour-initialised="true"] {
 
-.personalised-cover-download__icon {
-  filter: drop-shadow(0 0 0.75rem rgba(0,0,0,0.2));
+  .button-collection {
+    grid-row: 2;
+  }
+
+  .button-collection__item {
+    @include margin(12, "left");
+    @include margin(12, "right");
+  }
 }
 
-.personalised__button {
-  @include colors($color-primary);
-  @include padding(18 18 14);
-  font-weight: 500;
-  letter-spacing: 0.4px;
+ol.personalised-cover-buttons[data-behaviour-initialised="false"] {
```

Comment:
I believe this should be `.personalised-cover-buttons[data-behaviour-initialised="false"]`

Reply:
I agree and have moved them into .personalised-cover-buttons class to reduce the code. Works for me so please can you test?